<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>porto</class>
 <widget class="QDialog" name="porto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>369</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Anagrafica categorie</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Categoria</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Cerca</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="search"/>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="comboBox">
            <item>
             <property name="text">
              <string>Cerca per categoria</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Cerca per ID</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="cerca_categ">
            <property name="text">
             <string>Cerca</string>
            </property>
            <property name="icon">
             <iconset resource="../fabaria.qrc">
              <normaloff>:/images/edit-find.png</normaloff>:/images/edit-find.png</iconset>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>125</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="1">
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>125</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="1" colspan="2">
           <widget class="QLineEdit" name="id">
            <property name="styleSheet">
             <string notr="true">background-color:rgb(255, 255, 127)</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Categoria</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
          <item row="2" column="0">
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>58</width>
              <height>155</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="1">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>155</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="2">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>ID</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
      <zorder>label</zorder>
      <zorder>groupBox</zorder>
      <zorder>groupBox_3</zorder>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Lista categoria</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QTableView" name="tableView">
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>15</number>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="2">
       <widget class="QPushButton" name="salva">
        <property name="toolTip">
         <string>Salva</string>
        </property>
        <property name="text">
         <string>Salva</string>
        </property>
        <property name="icon">
         <iconset resource="../fabaria.qrc">
          <normaloff>:/images/document-save.png</normaloff>:/images/document-save.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="elimina">
        <property name="toolTip">
         <string>Elimina</string>
        </property>
        <property name="text">
         <string>Elimina</string>
        </property>
        <property name="icon">
         <iconset resource="../fabaria.qrc">
          <normaloff>:/images/edit-delete.png</normaloff>:/images/edit-delete.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="esci">
        <property name="toolTip">
         <string>Esci </string>
        </property>
        <property name="text">
         <string>Chiudi</string>
        </property>
        <property name="icon">
         <iconset resource="../../backup_restore_db/images.qrc">
          <normaloff>:/images/dialog-close.png</normaloff>:/images/dialog-close.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="nuovo">
        <property name="toolTip">
         <string>Nuovo</string>
        </property>
        <property name="text">
         <string>Nuovo</string>
        </property>
        <property name="icon">
         <iconset resource="../fabaria.qrc">
          <normaloff>:/images/filenew.png</normaloff>:/images/filenew.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>id</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>search</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>cerca_categ</tabstop>
  <tabstop>nuovo</tabstop>
 </tabstops>
 <resources>
  <include location="../../backup_restore_db/images.qrc"/>
  <include location="../fabaria.qrc"/>
 </resources>
 <connections/>
</ui>
